index.js

```diff
@@ -284,7 +284,7 @@ function buildExtraObjectPropertiesSerializer (context, location, addComma) {
       code += `
         if (/${propertyKey.replace(/\\*\//g, '\\/')}/.test(key)) {
           ${addComma}
-          json += serializer.asString(key) + ':'
+          json += serializer.asString(key,null) + ':'
```

Comment:
Why pass null? It’s type is object while the parameter type is string — to me that gives me the impression that an object argument is expected there

What’s wrong with leaving it as undefined?

```suggestion
          json += serializer.asString(key) + ':'
```